src/pages/index.js

```diff
@@ -18,7 +18,7 @@ class BlogIndex extends React.Component {
         const title = get(post, "node.frontmatter.title") || post.node.path
         pageLinks.push(
           <li
-            key={post.node.path}
+            key={post.node.fields.slug}
```

Comment:
We're not using custom slugs anymore. Change this to `post.node.frontmatter.path`.